includes/pull-ui.php

```diff
@@ -76,6 +76,13 @@ function setup_list_table() {
 		}
 	}
 
+	// Current user roles.
+	$current_user_roles = (array) wp_get_current_user()->roles;
+
+	// Check if user has cap to pull any external connection.
+	$pull_capabilities        = apply_filters( 'dt_pull_capabilities', 'manage_options' );
+	$can_pull_all_connections = current_user_can( esc_html( $pull_capabilities ) );
```

Comment:
I don't believe we need the `esc_html` here

Reply:
It was added to take care of the spam strings coming from the filter, removed for now.